samples/get_view_image.py

```diff
@@ -0,0 +1,67 @@
+####
+# This script demonstrates how to use the Tableau Server Client
+# to query a high resolution image of a view from Tableau Server.
+#
+# For more information, refer to the documentations on 'Query View Image'
+# (https://onlinehelp.tableau.com/current/api/rest_api/en-us/help.htm)
+#
+# To run the script, you must have installed Python 2.7.X or 3.3 and later.
+####
+
+import argparse
+import getpass
+import logging
+
+import tableauserverclient as TSC
+
+
+def main():
+
+    parser = argparse.ArgumentParser(description='Query View Image From Server')
+    parser.add_argument('--server', '-s', required=True, help='server address')
+    parser.add_argument('--username', '-u', required=True, help='username to sign into server')
+    parser.add_argument('--view-name', '-v', required=True, help='name of view')
+    parser.add_argument('--filepath', '-f', required=True, help='filepath to save the image returned')
+    parser.add_argument('--logging-level', '-l', choices=['debug', 'info', 'error'], default='error',
+                        help='desired logging level (set to error by default)')
+
+    args = parser.parse_args()
+
+    password = getpass.getpass("Password: ")
+
+    # Set logging level based on user input, or error by default
+    logging_level = getattr(logging, args.logging_level.upper())
+    logging.basicConfig(level=logging_level)
+
+    # Step 1: Sign in to server.
+    tableau_auth = TSC.TableauAuth(args.username, password)
+    server = TSC.Server(args.server)
+    server.version = 2.5
```

Comment:
This will just fail if run against a new Server (actually, it will fail on sign in).

My PR for detecting the version is outstanding, #100 -- waiting for approval to check it in.

But even then it wouldn't know which calls each version can and cannot make.

@RussTheAerialist had an idea to decorate the calls in the API with what version they use so we could throw a nice "NotSupported" exception or something, that work isn't yet started that I'm aware of (and is sorta dependent on my PR to get the version from Server)